chart/env/common.go

```diff
@@ -15,18 +15,42 @@ const (
 	// EnvVarKeepResources is the process environment variable representing the
 	// KEEP_RESOURCES env var.
 	EnvVarKeepResources = "KEEP_RESOURCES"
+	// EnvVarE2EKubeconfig is the process environment variable representing the
+	// E2E_KUBECONFIG env var.
+	EnvVarE2EKubeconfig = "E2E_KUBECONFIG"
```

Comment:
Nit but the constants should be in alpha order.

Reply:
Well spotted, thanks. Fixed it.